docs/setup_guides/cloud.rst

```diff
@@ -190,156 +147,59 @@ you can do so by running::
-to authenticate users. The ``clientSecret`` token needs to be obtained via
-github.
+Replace `<SECRET>` in the `secretToken` section with the output of this (note:
+you will stilll need the quotes around this value)
 
-This authentication method needs an IP or domain name to work, the IP you've
-reserved above and put in jupyter_config.yaml if you don't have a domain name
-yet (just put the IP in place of pangeo.pydata.org domain name).
 
-Alternatively, you can also change authentication method, see
-`Zero to Jupyterhub`_ guide for more information on that.
+Pangeo.pydata.org uses `GitHub OAuth Callback`_, (or `GitHub OAuth for developers`_)
+to authenticate users.
 
-.. code-block:: yaml
+This authentication method needs an IP or domain name to work. This should be
+the IP you've reserved above, if you don't have a domain name yet. Insert
+this IP in the following block::
 
-  # file: secret_config.yaml
+  proxy:
+      service:
+        loadBalancerIP: <GCE_EXTERNAL_IP>
 
-  jupyterhub:
-    proxy:
-      secretToken: <SECRET>
+Instead of `GCE_EXTERNAL_IP`.
 
-    # comment this out if not using github authentication
-    auth:
-      type: github
-      github:
-        clientId: "2cb5e09d5733ff2e6ae3"
-        clientSecret: <SECRET>
-        callbackUrl: "http://pangeo.pydata.org/hub/oauth_callback"
-      admin:
-        access: true
-        users:
-          - mrocklin
-          - jhamman
-          - rabernat
-          - yuvipanda
-          - choldgraf
-          - jacobtomlinson
+Replace `clientId`, `clientSecret` and `callbackUrl` values in the
+`secret_config.yaml` file with the values obtained from the GitHub app that you
+create for this.
 
+Alternatively, you can also change authentication method, see the
+`Zero to Jupyterhub`_ guide for more information on that.
 
 Step Five: Deploy Helm Chart
 ----------------------------
 
-The following script deploy the last Pangeo chart on your Kubernetes cluster.
+The following script deploys the most recent Pangeo Helm chart to your
+Kubernetes cluster.
 
 If you want to use a specific version, check `Pangeo Helm Chart
 <https://pangeo-data.github.io/helm-chart/>`_ for the version you want.
 You can then add a ``--version=0.1.1-a14d55b`` argument to ``helm install``
-command, only keeping the last part of the realease, without ``pangeo-v``.
-
-This script is available as ``3_deploy_helm.sh`` in the repo.
-
-.. code-block:: bash
-
-  #!/bin/bash
-
-  set -e
-
-  helm repo add pangeo https://pangeo-data.github.io/helm-chart/
-  helm repo update
-
-  helm install pangeo/pangeo --namespace=pangeo --name=jupyter \
-     -f secret_config.yaml -f jupyter_config.yaml
-
-  # helm install pangeo/pangeo --namespace=pangeo --name=jupyter \
-  #   --version=0.1.1-a14d55b \
-  #   -f secret_config.yaml -f jupyter_config.yaml
+command, only keeping the last part of the release, without ``pangeo-v``.
```

Comment:
suggest adding a "for example: ..." bit here. I'm not sure what is meant by the last part.